csv_detective/detect_fields/FR/geo/code_fantoir/__init__.py

```diff
@@ -1,7 +1,10 @@
-from frformat import CodeFantoir
+from frformat import CodeFantoir, Options
 
 PROPORTION = 1
 
-def _is (val):
+_code_fontoir = CodeFantoir(Options())
```

Comment:
```suggestion
_code_fantoir = CodeFantoir(Options())
```
y compris en dessous